{"name": "test_registration_with_invalid_data", "status": "passed", "start": 1722048363079, "stop": 1722048364047, "uuid": "f4701750-518b-43c9-bc2b-97946a676a59", "historyId": "20abe179da2ce0e7e7aabd1e3acbd83e", "testCaseId": "20abe179da2ce0e7e7aabd1e3acbd83e", "fullName": "test.test_Registration.TestLogin#test_registration_with_invalid_data", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_Registration"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Raunak"}, {"name": "thread", "value": "2884-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_Registration"}]}